apps/sms/style/sms.css

```diff
@@ -1038,6 +1038,26 @@ html[dir="rtl"] .contact-header aside.pack-end {
   float: left;
 }
 
+/* ellipsis direction hack before bug 883884 fixed */
+.ellipsis-dir-fix {
+  display: inline-block;
+  max-width: 100%;
+
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+/* margin and line-height adjustment for inline-block bdi element */
+.suggestion > .name > .ellipsis-dir-fix {
+  line-height: 2.6rem;
+  margin: -0.5rem 0;
+}
+
+.contact-header > .name > .ellipsis-dir-fix {
+  line-height: 3.2rem;
```

Comment:
You're right, I think we could update the existing rule instead to avoid the word cut.